src/add-to-app/android/project-setup.md

```diff
@@ -41,64 +41,85 @@ android {
   the Flutter module still runs correctly.
 {{site.alert.end}}
 
-## Using Android Studio
+## Integrate your Flutter module
 
-The Android Studio IDE is a convenient way of integrating
-your Flutter module automatically. With Android Studio,
-you can co-edit both your Android code and your Flutter code
-in the same project. You can also continue to use your normal
-IntelliJ Flutter plugin functionalities such as Dart code
-completion, hot reload, and widget inspector.
+{% comment %} Nav tabs {% endcomment -%}
```

Comment:
The section feels like a large formatting departure from the existing md file. Can this be written in a more plain markdown style?

Reply:
We use tabs elsewhere when a distinct choice is offered and it would reduce page length. This applies. The only difference is the tabs.